dydx/dydxViews/dydxViews/_v4/Vault/History/dydxVaultHistoryItemView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  dydxVaultHistoryItemView.swift
+//  dydxUI
+//
+//  Created by Rui Huang on 31/10/2024.
+//  Copyright © 2024 dYdX Trading Inc. All rights reserved.
+//
+
+import SwiftUI
+import PlatformUI
+import Utilities
+
+public class dydxVaultHistoryItemViewModel: PlatformViewModel {
+    @Published public var date: String?
+    @Published public var time: String?
+    @Published public var action: String?
+    @Published public var amount: String?
+    @Published public var onTapAction: (() -> Void)?
+
+    public init() { }
+
+    public static var previewValue: dydxVaultHistoryItemViewModel {
+        let vm = dydxVaultHistoryItemViewModel()
+        vm.date = "202-10-31"
```

Comment:
nit: invalid preview date?

Reply:
Yeah.. just for layout preview.. the content doesn;t matter

Reply:
Yep I assumed that. just wanted to flag 😛